retro_data_structures/properties/base_property.py

```diff
@@ -35,6 +43,36 @@ def from_json(cls: typing.Type[Self], data: typing.Any) -> Self:
 
     def to_json(self) -> typing.Any:
         raise NotImplementedError()
+    
+    def _is_property_mrea_or_mlvl(self, field: dataclasses.Field) -> bool:
+        return (('asset_types' in field.metadata)
```

Comment:
Probably could be simpler by doing `asset_types = field.metadata.get("asset_types", [])`?